src/components/TableHeadCell.js

```diff
@@ -116,11 +116,14 @@ const TableHeadCell = ({
     item: {
       type: 'HEADER',
       colIndex: index,
+      headCellRefs: draggableHeadCellRefs,
     },
     begin: monitor => {
-      setHintTooltipOpen(false);
-      setSortTooltipOpen(false);
-      setDragging(true);
+      setTimeout(() => {
```

Comment:
Can you tell what this is❓ 

why use setTimeout..

ref : https://react-dnd.github.io/react-dnd/docs/api/use-drag

begin(monitor): Optional. Fired when a drag operation begins. Nothing needs to be returned, but if an object is returned it will override the default item property of the spec.